app/build.gradle

```diff
@@ -58,6 +58,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-firestore-ktx'
     implementation 'com.firebaseui:firebase-ui-auth:7.1.1'
     implementation 'com.google.android.gms:play-services-maps:17.0.0'
+    implementation 'com.facebook.android:facebook-android-sdk:8.+'
```

Comment:
Do you think there's a way to make it work without using the + ? Cirrus doesn't seem happy about it...